src/js/main.js

```diff
@@ -0,0 +1,69 @@
+import "es5-shim";
```

Comment:
Add a `'use strict;'` to the top of each js file?

Reply:
I've created several modules now.. import is always strict, so we don't have to add it everywhere..